docs/src/model_creation/reactionsystem_content_accessing.md

```diff
@@ -0,0 +1,249 @@
+# [Accessing model properties](@id model_accessing)
+Catalyst is based around the creation, analysis, and simulation of chemical reaction network models. Catalyst stores these models in [`ReactionSystem](@ref) structures. This page describes some basic functions for accessing the content of these structures. This includes retrieving lists of species, parameters, or reactions that a model consists of. An extensive lists of relevant functions for working with `ReactionSystem` models can be found in Catalyst's [API](@ref ref).
+
+!!! warn
+    Generally, the field of Julia structures can be accessed through `struct.field`. E.g. a simulation's time vector can be retrieved using `simulation.t`. While Catalyst `ReactionSystem`s are structures, one should *never* access their fields using this approach, but rather using the accessor functions described below and in the [API](@ref ref) (direct accessing of fields can yield unexpected behaviours). E.g. to retrieve the species of a `ReactionsSystem` `rs`, use `Catalyst.get_species(rs)`, *not* `rs.species`.
+
+## [Direct accessing of symbolic model parameter and species](@id model_accessing_symbolic_variables)
```

Comment:
I think it is good to have a dedicated section going through this, and I think this is a place where it makes sense to put it (the other alternatives would be the advanced introduction or the compositional modelling part, right now I feel this was the most fitting place).